Application/EdFi.Ods.AdminApi.DBTests/ClaimSetEditorTests/GetAuthStrategiesByApplicationNameQueryTests.cs

```diff
@@ -8,65 +8,65 @@
-        };
-        Save(yetAnotherApplication);
-
-        var testApplication = new Application
-        {
-            ApplicationName = "TestApplicationName"
-        };
-        Save(testApplication);
-
-        SetupApplicationAuthorizationStrategies(otherApplication);
-        SetupApplicationAuthorizationStrategies(yetAnotherApplication);
-
-        var authStrategies = SetupApplicationAuthorizationStrategies(testApplication);
-
-        using var securityContext = TestContext;
-
-        var query = new GetAuthStrategiesByApplicationNameQuery(securityContext);
-        var results = query.Execute(authStrategies.First().Application.ApplicationName).ToArray();
-
-        results.Length.ShouldBe(authStrategies.Count);
-        results.Select(x => x.AuthStrategyName).ShouldBe(authStrategies.Select(x => x.AuthorizationStrategyName), true);
-    }
+    //[Test]
+    //public void ShouldGetAllTheAuthorizationStrategiesFromAnApplication()
+    //{
+    //    var testApplication = new Application
+    //    {
+    //        ApplicationName = "TestApplicationName"
+    //    };
+    //    SaveAdminContext(testApplication);
+
+    //    var authStrategies = SetupApplicationAuthorizationStrategies(testApplication);
+
+    //    using var securityContext = TestContext;
+    //    var query = new GetAuthStrategiesByApplicationNameQuery(securityContext);
+    //    var results = query.Execute(authStrategies.First().Application.ApplicationName).ToArray();
+
+    //    results.Length.ShouldBe(authStrategies.Count);
+    //    results.Select(x => x.AuthStrategyName).ShouldBe(authStrategies.Select(x => x.AuthorizationStrategyName), true);
+
+    //}
+
+    //[Test]
+    //public void ShouldNotGetAuthStrategiesFromOtherApplications()
+    //{
+    //    var otherApplication = new Application
+    //    {
+    //        ApplicationName = "OtherApplication"
+    //    };
+    //    Save(otherApplication);
+
+    //    var yetAnotherApplication = new Application
+    //    {
+    //        ApplicationName = "YetAnotherApplication"
+    //    };
+    //    Save(yetAnotherApplication);
+
+    //    var testApplication = new Application
+    //    {
+    //        ApplicationName = "TestApplicationName"
+    //    };
+    //    SaveAdminContext(testApplication);
+
+    //    SetupApplicationAuthorizationStrategies(otherApplication);
+    //    SetupApplicationAuthorizationStrategies(yetAnotherApplication);
+
+    //    var authStrategies = SetupApplicationAuthorizationStrategies(testApplication);
+
+    //    using var securityContext = TestContext;
+
+    //    var query = new GetAuthStrategiesByApplicationNameQuery(securityContext);
+    //    var results = query.Execute(authStrategies.First().Application.ApplicationName).ToArray();
+
+    //    results.Length.ShouldBe(authStrategies.Count);
+    //    results.Select(x => x.AuthStrategyName).ShouldBe(authStrategies.Select(x => x.AuthorizationStrategyName), true);
+    //}
```

Comment:
@jagudelo-gap 
Why these tests are commented ?

Reply:
I will remove this class because the file GetAuthStrategiesByApplicationNameQuery was removed due to being unused and these tests were for that class. So it's not needed anymore